rasa/validator.py

```diff
@@ -153,7 +153,7 @@ def verify_utterances(self, ignore_warnings: bool = True) -> bool:
 
         for action in actions:
             if action.startswith(UTTER_PREFIX):
-                if action not in utterance_templates:
+                if action not in utterance_responses:
                     rasa.shared.utils.io.raise_warning(
                         f"There is no template for the utterance action '{action}'. "
```

Comment:
```suggestion
                        f"There is no response for the utterance action '{action}'. "
```

Reply:
same in the warning in line 158 to 161

Reply:
and line 196

Reply:
Is line 196 the `if event.action_name not in utterance_actions:` clause?

Reply:
I deleted `verify_utterances` as both 146-152 and 158-161 are deleted.